packages/augur-ui/src/modules/markets-list/selectors/markets-list.ts

```diff
@@ -66,13 +70,50 @@ export const selectAllOtherCategories = createSelector(
 
     if (categories && categories.length > 0) {
       const allOtherCategories = categories
-        .filter(category => !POPULAR_CATEGORIES
-        .includes(category))
+        .filter(category => !POPULAR_CATEGORIES.includes(category))
         .map(mapObject)
-        .sort((a, b) => a.count > b.count ? -1 : 1);
+        .sort((a, b) => (a.count > b.count ? -1 : 1));
 
       return allOtherCategories;
     }
     return null;
   }
 );
+
+export const selectMarketStats = createSelector(
```

Comment:
new getter on this PR https://github.com/AugurProject/augur/pull/3971

to get category stats, meant to be used for market creation templates, we can prob use it for market stats as well. lets chat

Reply:
excellent! i'll wire that up in another PR.